### PR TITLE
Add: Cart Wishlist Functionality

### PR DESCRIPTION
**Pull request by devsurajrai**
Added functionality to the cart and Wishlist. Please review. 

LIVE LINK: https://phoenix-shop-r.netlify.app

cart
wishlist

### CODE REVIEW

File: src/customHooks/useCartWishlistData.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+      return cartWishlistState;
+  }
+};
+
+export const useCartWishlistData = () => {
+  const [cartWishlistState, cartWishlistStateDispatch] = useReducer(
+    reducer,
+    INITIAL_CART_WISHLIST_DATA
+  );
+  console.log(cartWishlistState);
```
Comment (palash23jain) [inline src/customHooks/useCartWishlistData.js:53]: you can remove console.log before deploying

File: src/backend/db/categories.js [hunk 2]
```diff
@@ -9,7 +9,8 @@ export const categories = [
   {
     _id: uuid(),
     categoryName: "racket",
-    categoryImgUrl: "https://shop-phoenix.netlify.app/Assets/images/racket.png",
+    categoryImgUrl:
```
Comment (palash23jain) [inline src/backend/db/categories.js:12]: you can prefer using webp or svg images for better performance

**Review by palash23jain — APPROVED** [review]
Overall coding looks clean. Good to go

**Comment by saifudinbadani** [comment]
Easy and simple to understand.. Maybe change class to class Name in rest of the elements so it won't give any warning.

**Pull request merged**